docs/DropdownToolbar.md

```diff
@@ -0,0 +1,61 @@
+# Dropdown Toolbar
+The `<DropdownToolbar>` component is used to display a toolbar with a dropdown.
```

Comment:
```suggestion
The `<DropdownToolbar>` component is used to display a toolbar with a dropdown menu for a subtitle.
```

Reply:
May also want to specify something about you can pass in a list of menuitems to render or your own custom menu content if you want more control.